spec/cookbook/documentation_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec_helper'
+require 'ostruct'
+
+link_lines = `egrep -R 'https:\/\/github.com\/sous-chefs\/line' *`.split("\n")
+line_files = link_lines.map do |line|
+  md = %r{\]\(https://github.com/sous-chefs/line/blob/master/(.*)\)}.match(line)
+  md ? md[1] : nil
+end
+describe 'linked documentation files exist' do
+  line_files.compact.each do |file|
+    it "File #{file} should exist" do
+      expect(File.exist?(file)).to eq true
+    end
+  end
+end
```

Comment:
This seems a little OTT, but 

![giphy](https://user-images.githubusercontent.com/40786/56344496-7f0c5400-61b5-11e9-9751-7c7fcff452aa.gif)

Reply:
I sort of think so too. The first error we encountered with putting documentation into a separate directory was that the files weren't there.  Changing to use the links to the master branch means we'll miss accidental file deletes or moves.  So I'm checking.

Reply:
I wonder if we cna move this test out of the cookbook, and into the circleci pipeline so we can all reuse it?

For now, this is just an ace example

Reply:
Let's see if this actually fixes the problem in supermarket.  Cheers